config/all-in-one/values.yaml

```diff
@@ -89,6 +92,7 @@ cp4waiops:
     instanceName: aiops-installation
     # The namespace where AI Manager is installed.
     namespace: cp4waiops
+    # channel: v3.3
```

Comment:
This may need to be changed to v3.4 since it's going to be merged to 3.4 branch.

Reply:
Agree and will update later.